src/components/CryptoTraq.js

```diff
@@ -1,25 +1,38 @@
-import React, { Component } from 'react';
-import CompareButton from './CompareButton';
+/*
+		Did some cleaning up of imports in this component and added Radium for the arrow
+		styling.
+		This component had some refactoring when I made the SelectionForm component to
+		break up the big form component I had before.  Mainly adding the new components
+		and changing the props that are passed in accordingly.
+		Button component is also used here now instead of CompareButton.
+ */
+
+import React from 'react';
+import Radium from 'radium';
+import Button from './Button';
+import SelectionForm from './SelectionForm';
 import CryptoSelectForm from './CryptoSelectForm';
+import IntlSelectForm from './IntlSelectForm';
 import ExchangeResult from './ExchangeResult';
 import Graph from './Graph';
-import IntlSelectForm from './IntlSelectForm';
-import SupportedCurrencies from './SupportedCurrencies';
```

Comment:
Did some cleaning up of imports in this component and added Radium for the arrow styling.